### PR TITLE
Restructures and expands 'Publish to SPFx marketplace' docs

### PR DESCRIPTION
**Pull request by waldekmastykarz**
## Category

- [ ] Content fix
- [x] New article

## What's in this Pull Request?

Restructures and expands 'Publish to SPFx marketplace' docs

### CODE REVIEW

File: docs/spfx/publish-to-marketplace-faq.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: Publish SharePoint Framework applications to the marketplace - FAQ
+description: Frequently asked questions about publishing SharePoint Framework applications to the marketplace
+ms.date: 06/25/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# Frequently asked questions
+
+## Is it required to configure web parts to be isolated for publishing in the marketplace
```
Comment (andrewconnell) [inline docs/spfx/publish-to-marketplace-faq.md:11]: Issue: don't start a doc with two headings.

- Docs must start with an H1 that matches the doc title
- Docs should have an overview statement between the title & the first subheading

File: docs/spfx/publish-to-marketplace-faq.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Publish SharePoint Framework applications to the marketplace - FAQ
+description: Frequently asked questions about publishing SharePoint Framework applications to the marketplace
+ms.date: 06/25/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# Frequently asked questions
```
Comment (andrewconnell) [inline docs/spfx/publish-to-marketplace-faq.md:9]: Should match the doc title

File: docs/spfx/publish-to-marketplace-checklist.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
+---
+title: Prepare your SharePoint Framework application for publishing to the marketplace
+description: Tips & tricks to help you get your SharePoint Framework application published in the marketplace
+ms.date: 06/25/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# App validation checklist
```
Comment (andrewconnell) [inline docs/spfx/publish-to-marketplace-checklist.md:9]: Should match the document title... the H1 value is what is shown in the page itself. The `title` metadata property is used in the <HTML><HEAD></></> section.

File: docs/spfx/publish-to-marketplace-after-publishing.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+title: Maintain and support your published app
+description: Tips to maintain and support your SharePoint Framework app after publishing it to the marketplace
+ms.date: 06/25/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# Maintain and support your published app
+
+## Install applications from the marketplace in the tenant app catalog
```
Comment (andrewconnell) [inline docs/spfx/publish-to-marketplace-after-publishing.md:11]: Issue: don't start a doc with two headings.

- Docs should have an overview statement between the title & the first subheading

**Comment by waldekmastykarz** [comment]
Thanks for the review @andrewconnell. All issues are addressed in an additional commit

**Comment by VesaJuvonen** [comment]
thx @waldekmastykarz  and @andrewconnell!

**Pull request merged**